build.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
```

Comment:
What about applying this changes in `docker-compose.yml`, would it makes more sense? Like this:

```
platform: linux/amd64
```

Reply:
let me check if it has the same result, but quite probably yours is the better approach

Reply:
works like a charm